classes/02class/exercises/c02-network08/frdvo/README.md

```diff
@@ -0,0 +1,289 @@
+
+```
+$aws ec2 describe-network-acls
+{
+    "NetworkAcls": [
+        {
+            "Associations": [
+                {
+                    "NetworkAclAssociationId": "aclassoc-02e4d0fd9669146d2",
+                    "NetworkAclId": "acl-0e036e5c5c674f74b",
+                    "SubnetId": "subnet-074a113f80e846338"
+                },
+                {
+                    "NetworkAclAssociationId": "aclassoc-0e90bff240e45eae1",
+                    "NetworkAclId": "acl-0e036e5c5c674f74b",
+                    "SubnetId": "subnet-01f6ba32065d77e19"
+                },
+                {
+                    "NetworkAclAssociationId": "aclassoc-060ff6113416e5e61",
+                    "NetworkAclId": "acl-0e036e5c5c674f74b",
+                    "SubnetId": "subnet-0b9349fa56d76afb7"
+                },
+                {
+                    "NetworkAclAssociationId": "aclassoc-6c6ff511",
+                    "NetworkAclId": "acl-0e036e5c5c674f74b",
+                    "SubnetId": "subnet-04843486979b4aeb2"
+                },
+                {
+                    "NetworkAclAssociationId": "aclassoc-0ffb88c932c7e379c",
+                    "NetworkAclId": "acl-0e036e5c5c674f74b",
+                    "SubnetId": "subnet-0ba23989d8f1b0768"
+                },
+                {
+                    "NetworkAclAssociationId": "aclassoc-0afb3136eee16f933",
+                    "NetworkAclId": "acl-0e036e5c5c674f74b",
+                    "SubnetId": "subnet-0dbb2205880974071"
+                }
+            ],
+            "Entries": [
+                {
+                    "CidrBlock": "0.0.0.0/0",
+                    "Egress": true,
+                    "Protocol": "-1",
+                    "RuleAction": "allow",
+                    "RuleNumber": 100
+                },
+                {
+                    "CidrBlock": "0.0.0.0/0",
+                    "Egress": true,
+                    "Protocol": "-1",
+                    "RuleAction": "deny",
+                    "RuleNumber": 32767
+                },
+                {
+                    "CidrBlock": "0.0.0.0/0",
+                    "Egress": false,
+                    "Protocol": "-1",
+                    "RuleAction": "allow",
+                    "RuleNumber": 100
+                },
+                {
+                    "CidrBlock": "0.0.0.0/0",
+                    "Egress": false,
+                    "Protocol": "-1",
+                    "RuleAction": "deny",
+                    "RuleNumber": 32767
+                }
+            ],
+            "IsDefault": true,
+            "NetworkAclId": "acl-0e036e5c5c674f74b",
+            "Tags": [],
+            "VpcId": "vpc-0a2b7db4956438d22",
+            "OwnerId": "165765640813"
+        }
+    ]
+} 
+ 
+$aws ec2 create-network-acl-entry --network-acl-id acl-0e036e5c5c674f74b --ingress --rule-number 200 --protocol tcp --port-range From=22,To=22 --cidr-block 172.16.200.0/24 --rule-action deny
```

Comment:
As you have an ingress nacl that allow everything under number `100`. Your nacl that blocks something need to have a lower number than that.